Perf map docs (#29102)
dotnet/docs: docs/core/runtime-config/debugging-profiling.md

| **Environment variable** | `CORECLR_PROFILER_PATH_32` | *string-path* |
| **Environment variable** | `CORECLR_PROFILER_PATH_64` | *string-path* |

## Export perf maps

- Enables or disables emitting perf maps to */tmp/perf-$pid.map*. Perf maps allow 3rd party tools, such as perf, to identify call sites from pre-compiled R2R modules.
- If you omit this setting, writing the perf map is disabled. This is equivalent to setting the value to `0`.

> - Without perf maps enabled you will not see all managed callsites properly resolved.

Reviewer: ```suggestion
- When perf maps are disabled, not all managed callsites will be properly resolved.
```

> - Enabling causes a 10-20% overhead.

Reviewer: ```suggestion
- Enabling perf maps causes a 10-20% overhead.
```
